### PR TITLE
Deploy two

### PR DESCRIPTION
**Pull request by thebinarypenguin**
the server may still crash with bad input

### CODE REVIEW

**Review by ZolFallows — APPROVED** [review]
Looks good

File: seeds/seed_data.sql [hunk 1]
```diff
@@ -125,37 +127,39 @@ INSERT INTO "incomes" ("id", "owner_id", "category_id", "description", "amount",
...
   VALUES
     (
         1,
         1,
         1,
         'Rent',
         1200.00,
-        'DTSTART:20181001T000000Z
-RRULE:FREQ=MONTHLY;INTERVAL=1;WKST=MO'
+        make_timestamptz(2019, 1, 1, 0, 0, 0),
```
Comment (cjcarnivale) [inline seeds/seed_data.sql:144]: What do these changes do?

Comment (thebinarypenguin) [reply seeds/seed_data.sql:144]: Breaking the recurring rule and start dat out into own columns

**Review by ghost — APPROVED** [review]
good to deploy!!

**Review by cjcarnivale — APPROVED** [review]

**Pull request merged**